.github/workflows/tag-release.yml

```
@@ -0,0 +1,36 @@
+jobs:
+  tag:
+    name: Create tag and draft release
+    if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.title, 'Bump version to')
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Extract version from PR title
+        run: |
```

actionlint (1.7.11)

[error] "github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)